### PR TITLE
Updated version and dependencies

### PR DESCRIPTION
**Pull request by Horusiath**
TODOs:

- [x] **Remove FsPickler** - it won't be needed, Hyperion can handle expressions for remote deployments.
- [x] **Update all dependencies** (akka.net v1.3.3, explicitly added dep on Microsoft.Extensions.Logging.Abstractions)
- [x] **Add Paket fsx load scripts** for sake of sample files - since dependency on DotNetty at some point introduced dozens of new libs to be referenced in fsx files, we need to simplify that process.
- [x] **Update sample files** - I think, they are not working as expected from quite a while.
- [x] **Update fsproj files to target both net452 and netstandard1.6** - one of the blockers for .NET Core support in the past was weak support for it by F# itself and fact, that FsPickler still lags behind. It's no longer the case.

### CODE REVIEW

File: tests/Akkling.Tests/app.config [hunk 1]
```diff
@@ -53,6 +53,16 @@
     <assemblyIdentity name="Google.Protobuf" publicKeyToken="a7d26565bac4d604" culture="neutral" />
     <bindingRedirect oldVersion="0.0.0.0-65535.65535.65535.65535" newVersion="3.5.1.0" />
   </dependentAssembly>
+  <dependentAssembly>
+    <Paket>True</Paket>
+    <assemblyIdentity name="Microsoft.Extensions.Logging" publicKeyToken="adb9793829ddae60" culture="neutral" />
+    <bindingRedirect oldVersion="0.0.0.0-65535.65535.65535.65535" newVersion="1.1.2.0" />
```
Comment (vasily-kirichenko) [inline tests/Akkling.Tests/app.config]: Hmm. I did not managed to make it work with any version other than 1.1.1.0.

Comment (Horusiath) [reply tests/Akkling.Tests/app.config]: I've unlocked tests checking remote deployment scenario between two actor systems communicating over autentic DotNetty transport. It passed.

**Comment by Horusiath** [comment]
One blocker found: FSharp.Quotations.Evaluator doesn't support .NET Standard.

**Comment by Horusiath** [comment]
I think, I'll try to add conditional regions to make .NET Standard working for now - i.e. keep `Expr`-based API on .NET Framework 4.5, but drop it on .NET Standard 1.6. This will be backward compatible. 

I only need to set up FSharp.Quotations.Evaluator as dependency on v4.5 but not on v1.6 of netstandard.

**Comment by Horusiath** [comment]
At this moment I'm running most of the stuff, but two concerns apply:

1. I had to remove the dependency on FSharp.Quotations.Evaluator. Essentially this means, that I had to remove every part of an API, which uses `Expr<>`. It's a breaking change, mostly to be mitigated by a code upgrade.
2. I'm getting `internal error: convMethodRef: could not bind to method` when trying to run examples folder files in FSI. No idea yet, what's causing this.

**Comment by Horusiath** [comment]
@cotyar could you make the PR? I'm not able to make it work on loading main group too.

**Comment by cotyar** [comment]
Strange it worked last night but doesn't today. 
But never mind, I've found the reason for the issue I think. 
If send to Interactive everything from top until the end of "let behavior ..." if gives the reason of failure: 
"Binding session to 'C:\Program Files (x86)\Microsoft SDKs\F#\4.1\Framework\v4.0\FSharp.Core.dll'..."
On my machine this file is dated "04 ‎November ‎2017" but the one in Akkling folder (used for compilation) is from "09 ‎February ‎2018". File sizes are also different. 

Not sure how to make the FSI to pick up the right one so ...

**Comment by cotyar** [comment]
Trying to add explicitly:
#r "../src/Akkling/bin/Release/net452/FSharp.Core.dll"
Getting this:
error FS1222: When mscorlib.dll or FSharp.Core.dll is explicitly referenced the --noframework option must also be passed

**Comment by cotyar** [comment]
Removing DSharp.Core from paket.dependencies/paket.references breaks the build
Adding "FSharp.fsiExtraParameters": "--noframework" to Ionide config prevents F# interactive from starting ...

**Pull request merged**